### PR TITLE
Aro migration api

### PR DESCRIPTION
**Pull request by patricksimonian**
## Summary
Devhub API migration

Fix some odds and ends in the infracode. Also updated the pipeline scripts to actually clean things up during the cleanup stage.

### CODE REVIEW

File: api/.pipeline/lib/clean.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ module.exports = (settings) => {
       });
       oc.raw(
         'delete',
-        ['pvc,Secret,configmap,endpoints,RoleBinding,role,ServiceAccount,Endpoints'],
+        ['pvc,Secret,configmap'],
```
Comment (patricksimonian) [inline api/.pipeline/lib/clean.js:68]: no rolebindings are created in namespaces for the api

File: api/.pipeline/lib/deploy.js [hunk 2]
```diff
@@ -77,4 +77,6 @@ module.exports = (settings) => {
   );
   oc.importImageStreams(objects, phases[phase].tag, phases.build.namespace, phases.build.tag);
   oc.applyAndDeploy(objects, phases[phase].instance);
+  // wait for roll out
+  oc.raw('rollout status', [`dc/${phases[phase].name}-${phases[phase].suffix}`], { w: true})
```
Comment (ShellyXueHan) [inline api/.pipeline/lib/deploy.js:81]: easier to just use ${phases[phase].instance}?

**Review by ShellyXueHan — APPROVED** [review]
I like the component specific label

File: api/.pipeline/lib/config.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ const name = 'devhub-api';
 
 const phases = {
   build: {
-    namespace: 'devhub-tools',
+    namespace: 'pltfrm-tools',
     name: `${name}`,
```
Comment (AndrewJun97) [inline api/.pipeline/lib/config.js:10]: This doesn't need to be templated. Ex// `name: name` :)

**Review by AndrewJun97 — APPROVED** [review]

**Pull request merged**